projects/plugins/jetpack/modules/shortcodes/recipe.php

```diff
@@ -493,7 +493,7 @@ private static function output_list_content( $content, $type ) {
 		if (
 			strpos( $content, '&#8211;' ) !== false ||
 			strpos( $content, '&#8212;' ) !== false ||
-			strpos( $content, '-' ) !== false ||
+			str_contains( $content, '-' ) ||
```

Comment:
Why only this one and not all the rest in here?

Reply:
Done in 141bf1195491aef909482be92b19a7d0563a7985